src/settings.js

```diff
@@ -1743,7 +1743,7 @@ var MIN_FIREFOX_VERSION = 65;
 // see https://github.com/emscripten-core/emscripten/pull/7191.
 // MAX_INT (0x7FFFFFFF, or -1) specifies that target is not supported.
 // [link]
-var MIN_SAFARI_VERSION = 120000;
+var MIN_SAFARI_VERSION = 150000;
```

Comment:
Isn't Safari 15 super recent, like July 2021? This would make Emscripten support only 1 year old Safari by default? That might cause a lot of surprises.

Reply:
I mean, I am not opposed to this change as long as we can manually and explicitly fall back, but for this kind of change, we should be super loud and highlight this change every place possible, so it is not a surprise that Emscripten output will no longer work on every browser and phone that supports Wasm.

Reply:
Yes, this one surprises me too. It's less than a year old if [wikipedia](https://en.wikipedia.org/wiki/Safari_(web_browser)#Safari_15) is correct.

The larger question is market share, I guess. I don't actually know how best to estimate that, but [caniuse/statcounter](https://caniuse.com/usage-table) suggests 15 is by far the most popular, but 14 still has a small not significant amount - maybe 1/5 to 1/10th of 15's share. That's enough to worry me.

Has this been taken into account on the LLVM side? If not maybe we can discuss that there.

Reply:
@dschuff are we sure that bulk-memory and non-trapping-float-to-int really depend on Safari 15?

Reply:
This is sourced from Safari's release notes: https://developer.apple.com/documentation/safari-release-notes/safari-15-release-notes